repl/src/main/scala/ammonite/repl/Storage.scala

```diff
@@ -20,6 +20,8 @@ trait Storage{
 
   val history: Ref[History]
   val ivyCache: Ref[IvyMap]
+  def compileCacheSave(tag: String, classFiles: Traversable[(String, Array[Byte])], imports: Seq[ImportData]): Unit
+  def compileCacheLoad(tag: String): Option[(Traversable[(String, Array[Byte])], Seq[ImportData])]
```

Comment:
Could we break out the `Traversable[(String, Array[Byte])]` into a type alias? We use it all over the place so having a short name for it would be nice